src/TextForEvent.tsx

```diff
@@ -635,7 +661,7 @@ export function hasText(ev): boolean {
     return Boolean(handler?.(ev));
 }
 
-export function textForEvent(ev): string {
+export function textForEvent(ev): string | JSX.Element {
```

Comment:
So textForEvent is also called upon by Notifier which definitely needs a string and not a JSX Element